packages/manager/modules/dbaas-logs/src/logs/detail/encryption-keys/logs-encryption-keys.controller.js

```diff
@@ -0,0 +1 @@
+export default class LogsEncryptionKeysCtrl {}
```

Comment:
**nitpick:** Remove the empty controller.

Reply:
Removed